recipes/h2o-py/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "h2o" %}
+{% set version = "3.26.0.2" %}
+
+package:
+  name: "{{ name|lower }}-py"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "839291a0f74b18790120ecc16343cede545dac668a953fc94455775022104dda"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
can this be noarch: python?

Reply:
@scopatz thanks for spotting this! fixed.